spyder_kernels/utils/nsview.py

```diff
@@ -558,7 +561,10 @@ def get_human_readable_type(item):
             return "Image"
         else:
             text = get_type_string(item)
-            return text[text.find('.')+1:]
+            if text == 'polars.dataframe.frame.DataFrame':
+                return 'Polars DataFrame'
+            else:
+                return text[text.find('.')+1:]
```

Comment:
```suggestion
                return text[text.find('.') + 1:]
```

Minor code style improvement.